doc/circuitikzmanual.tex

```diff
@@ -7027,6 +7028,27 @@ \subsubsection{Mux-Demux anchors}
 }
 \end{lstlisting}
 
+\subsubsection{Adding European-style inversion markers}\label{sec:eurinv}
+
+Although you can add ``negation balls'' as seen for, for example, flip-flops (see section~\ref{sec:flip-flop-cust}), sometimes the Europen-style notation with the small triangle is preferred. The \texttt{euroinv} shape will nicely do. It'll scale with the \texttt{muxdemuxes} class, and the length and height can be changed with the keys \texttt{european inversion mark/width} (default \texttt{0.2}) and \texttt{height} (default \texttt{0.1}), with the same units that are used for the \texttt{external pins width} and similar keys.
```

Comment:
1. I am forgetting if other node-style components have this or not, but how about sharing the "default anchor"?

Also, just would like this final opportunity to ponder over aptness of the name. Do you think it is correct and general enough - given the IEEE use of this triangle? `euroinv`? I think it is - the word invert is fairly genric word, applies partially well both to logic (invert the logic) as well as to polarity (reverse the polarity), and prepending "euro" gives good enough distinction i guess. But i don't have much experience.

Reply:
Well, the generic anchor is the one named `center` for every shape. You are right, I should add the `center` anchor to the description box, given that's in a non-standard position.
I'll add it!

Reply:
I think the default anchor here is the one at base
with default, i meant the one which's used when an anchor is not explicitly specified

Reply:
Yes, but in any Ti*k*Z shape, the default anchor is the one named `center`. In this case, it coincides with `base`, and now it's shown in the "component box".

Reply:
> but in any Ti_k_Z shape, the default anchor is the one named `center`. In this case, it coincides with `base`, and now it's shown in the "component box".

ohwww, ohkayyh. neat 👍